.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,39 @@
+name: Pull-Request
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+      - '[0-9].[0-9]+'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+
+jobs:
+  compile-deps:
+    name: Compile deps
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    uses: aio-libs/aiohttp/.github/workflows/compile-deps.yml@0c8a36f3383a4b27ba2facda47944f314d0cf8c2
+    secrets:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  # TODO: replace to @master after the PR merging
+  lint:
+    name: Lint
+    needs: compile-deps
+    uses: aio-libs/aiohttp/.github/workflows/lint.yml@0c8a36f3383a4b27ba2facda47944f314d0cf8c2
+  test:
+    name: Test
+    needs: lint
```

Comment:
I think this shouldn't be blocked on linting.

Reply:
Ok, agree